### PR TITLE
Qubes cb dev

### PR DESCRIPTION
**Pull request by pietrushnic**

### CODE REVIEW

File: post.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+I don't think much people will face so weird configuration, but I need below
+notes for myself and there is some chance that someone will face similar
+issues.
+
+To summarize my target configuration was like that:
+
+![qubes-apu2-setup](https://3mdeb.com/wp-content/uploads/2017/07/qubes-apu2-setup.png)
+
+My initial idea was to have servers on AppVMs, but I didn't have enough time to
+get through QubesOS `iptables` rules. What lead to discover interesting
```
Comment (miczyg1) [inline post.md]: That led

File: post.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+alternative with `proxychains`.
```
Comment (miczyg1) [inline post.md]: The thread is cut all of a sudden here. Maybe consider adding something like: "... proxychains, which I will describe later in this article".

File: post.md (hunk 1 above)
Comment (pietrushnic) [reply post.md]: ok

File: post.md (hunk 2 above)
Comment (pietrushnic) [reply post.md]: ok

**Comment by pietrushnic** [comment]
@mek-x any comments ?

File: post.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+  - networking
+  - pxe
+  - nfs
+  - iptables
+  - ssh
+categories:
+  - OS Dev
+---
+
+At some point I stuck in the forest with WiFi connection and no physical access
```
[the remark this answers is not in the document]
Comment (pietrushnic) [reply post.md]: @mek-x I would not say it is idiom, I wanted to say that I was in Bory Tucholskie ;P

**Comment by pietrushnic** [comment]
@mek-x @miczyg1 anything else I can fix ?

File: post.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (miczyg1) [reply post.md]: Well Bory Tucholskie are almost no more since last hurricane in the middle of August :P

**Comment by miczyg1** [comment]
Fair enough for me.

**Comment by mek-x** [comment]
Seems ok

**Pull request merged**